README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Practice importing data from CSV.

Used the abalone data set from here: http://archive.ics.uci.edu/ml/datasets/Abalone

Imported using:
Python core: csv.reader(pythonfile)
numpy: np.loadtxt('filepath', delimiter=',', etc...)
pandas: pd.read_csv - http://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html

Use 'converter' function to convert abalone rings to age by adding 1.5 as advised in the data set site.